src/problem1.rb

```diff
@@ -0,0 +1,17 @@
+class Problem1
+  def sumOfAllEvenNumbers(values)
+    response = 0
+    if(!values.nil? && (values.instance_of?(Array)))
```

Comment:
There is no need for this checks, the problem text is very clear that you can only receive a an array of integers

Reply:
This is usually consider a bad practice, read more about it here https://www.martinfowler.com/bliki/Yagni.html